src/aat/resources/features/F-142 - Get Linked Case V2 External/CreateCases/F-142_CreateLinkedCase_NonStandardCaseLink.td.json

```diff
@@ -0,0 +1,38 @@
+{
+  "_guid_": "F-142_CreateLinkedCase_NonStandardCaseLink",
+  "_extends_": "F-142_CreateCase_MasterCaseType__Base",
+
+  "request": {
+    "body": {
+      "data": {
+        "caseNameHmctsInternal": "Name internal: ${[scenarioContext][customValues][generateUUID]}",
```

Comment:
Does this need to be a UUID here given that we only check its a non null string in the response ?